Formula/xmlsectool.rb

```diff
@@ -0,0 +1,19 @@
+class Xmlsectool < Formula
+  desc "Check schema validity and signature of an XML document."
+  homepage "https://wiki.shibboleth.net/confluence/display/XSTJ2/xmlsectool+V2+Home"
+  url "https://shibboleth.net/downloads/tools/xmlsectool/latest/xmlsectool-2.0.0-bin.zip"
+  sha256 "83ea1e977d947dd5f2f5600a3280bc5d372a4913f0dbce1ddf2c0ca1c869ca57"
+
+  depends_on :java => "1.7+"
+
+  def install
+    rm_rf "doc"
```

Comment:
Couldn't hurt to `prefix.install "doc/LICENSE.TXT"` (we have an `#install_metafiles` method that does it automatically if it's in the directory root).